### ISSUE TITLE
File permissions in gem are incorrect

### ISSUE THREAD

**Issue by justindossey** [chrislee35/dnsbl-client#10]
It looks like everything in the gem is 0700, which means that only the user who installed the gem can use it. In application scenarios, it's is expected that the runtime user doesn't own the gems' files.

```$ tar tvf data.tar.gz
-rwx------ wheel/wheel     529 2022-07-03 16:53 bin/dnsbl-client
-rwx------ wheel/wheel    5294 2022-07-03 16:53 data/dnsbl.yaml
-rwx------ wheel/wheel   35171 2022-07-03 16:53 data/three-level-tlds
-rwx------ wheel/wheel   62955 2022-07-03 16:53 data/two-level-tlds
-rwx------ wheel/wheel   11047 2022-07-03 16:53 lib/dnsbl/client.rb
-rwx------ wheel/wheel     147 2022-07-03 16:53 lib/dnsbl/client/version.rb
-rwx------ wheel/wheel     141 2022-07-03 16:53 test/helper.rb
-rwx------ wheel/wheel    8909 2022-07-03 16:53 test/test_dnsbl-client.rb
```

**Comment by chrislee35** [comment]
Good catch.  Probably happened when I changed filesystems.  I'll get that fixed pronto.

**Comment by chrislee35** [comment]
I think it's fixed in 1.1.1 now.  My testing harness is suboptimal right now.
